sklearn/cluster/_agglomerative.py

```diff
@@ -747,6 +747,13 @@ class AgglomerativeClustering(ClusterMixin, BaseEstimator):
 
         .. versionadded:: 0.21
 
+    compute_distances : bool, default=False
+        Computes distances even if no distance_threshold is used. This can be
+        used to make dendrogram visualization, but has impact on efficiency due
+        to the additional computational cost and used memory.
+
+        .. versionadded:: 0.24
+
     Attributes
     ----------
```

Comment:
Can you please document the new `distances_` attribute?

Reply:
actually `distances_` is already documented but its docstring needs to be updated to document the interaction with the `compute_distances` parameter.

Reply:
The interaction with `compute_distances` is now documented.
Fixed in 61e5c5a.